### PR TITLE
Explain how to pass the configuration to the UDW

### PR DESCRIPTION
**Pull request by Plopix**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      |

### CODE REVIEW

File: docs/extending/extending_udw.md [hunk 1]
```diff
@@ -5,14 +5,28 @@ using an interactive interface: browse, search, create, and bookmarks view.
...
+
+> `simple` configuration is one of the default configuration provided. You can also do your [own configuration](#Adding-new-configuration).
+
 With vanilla JS:
 
 ``` js
 const container = document.querySelector('#react-udw');
 
+const config = /* fetch the config somewhere */;
+//const config = JSON.parse(somewhere.dataset.udwConfig);
```
Comment (dew326) [inline docs/extending/extending_udw.md:24]: I would write something like this:
```suggestion
//const config = JSON.parse(document.querySelector('.btn-udw-trigger).dataset.udwConfig);
```

And I would add this class to the button above to show which udwConfig should be passed to the React component

File: docs/extending/extending_udw.md [hunk 2]
```diff
@@ -5,14 +5,28 @@ using an interactive interface: browse, search, create, and bookmarks view.
...
+
 ReactDOM.render(React.createElement(eZ.modules.UniversalDiscovery, {
     onConfirm: {Function},
     onCancel: {Function},
+    {...config}
```
Comment (dew326) [inline docs/extending/extending_udw.md:29]:
```suggestion
    ...config
```

File: docs/extending/extending_udw.md [hunk 3]
```diff
@@ -5,14 +5,28 @@ using an interactive interface: browse, search, create, and bookmarks view.
...
 
+UDW requires configuration that you can provide using the `ez_udw_config` Twig helper. This configuration must be spread to the props of the component itself.
+
+```html+twig
+<button data-udw-config="{{ ez_udw_config('single') }}">
+    Open My UDW
+</button>
+```
+
+> `simple` configuration is one of the default configuration provided. You can also do your [own configuration](#Adding-new-configuration).
```
Comment (dew326) [inline docs/extending/extending_udw.md:16]: `simple` or `single`?

**Review by dew326 — CHANGES_REQUESTED** [review]

File: docs/extending/extending_udw.md [hunk 4]
```diff
@@ -5,14 +5,28 @@ using an interactive interface: browse, search, create, and bookmarks view.
 
 ## How to use UDW?
 
+UDW requires configuration that you can provide using the `ez_udw_config` Twig helper. This configuration must be spread to the props of the component itself.
```
Comment (dabrt) [inline docs/extending/extending_udw.md:8]:
```suggestion
UDW requires that you provide configuration by using the `ez_udw_config` Twig helper. This configuration must be spread to the properties of the component itself.
```
Also - what do you mean by "spread" - is it "distributed" or "populated"?

**Review by dabrt — CHANGES_REQUESTED** [review]

**Review by dew326 — APPROVED** [review]

**Review by dabrt — APPROVED** [review]

**Pull request merged**